target/linux/ramips/dts/mt7621_adslr_g7.dts

```diff
@@ -105,13 +105,41 @@
 	};
 };
 
-&ethernet {
+&gmac0 {
 	mtd-mac-address = <&factory 0xe00c>;
 };
 
+&switch0 {
+	ports {
+		port@0 {
+			status = "okay";
+			label = "lan1";
+		};
```

Comment:
Since this is setting the blueprint for the future, consider the port nodes should be separated by empty lines like other nodes or not. I do not have a preference at the moment.

Reply:
The port nodes from upstream `drivers/staging/mt7621-dts` are not separated

Reply:
Okay, so let's just keep it as you did it.